bench/index.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+var benchmark = require('./benchmark');
+
+var Identity = require('../test/Identity');
+
+//  inc :: Number -> Number
+function inc(x) {
+  return x + 1;
+}
+
+benchmark({
+  warmup:                        function()  { Identity(inc(253)); },
```

Comment:
When aligning text at a particular column I like to use a multiple of two—preferably a multiple of four—because I like even numbers (as does my text editor). :)

We're currently at 33. Let's use 36 or 40 instead. This will also reduce the likelihood of needing to shift the column at some point in the future to accommodate a longer key.